app/decorators/staff/time_slot_decorator.rb

```diff
@@ -81,31 +77,23 @@ def linked_title
   end
 
   def display_presenter
-    if object.program_session.present?
-      speaker_names(object.program_session)
-    else
-      object.presenter
-    end
+    speaker_names(object.program_session) || object.presenter
```

Comment:
I prefer the original syntax that explicitly checks if the program session is set vs the magic of leaning on try/||.  The new approach is more concise in a single line but we lose the readability and distinction of why we're falling back to the other value.  In this case it appears to unnecessarily execute code when a program session isn't set.